extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.css

```diff
@@ -40,4 +40,124 @@ amp-story-subscriptions.i-amphtml-story-subscriptions-visible::before {
 amp-subscriptions-dialog.i-amphtml-story-subscriptions-dialog-wrapper {
   border-radius: 16px 16px 0px 0px !important;
   font-family: Poppins, sans-serif !important;
+  background: white !important;
+  color: black !important;
+}
+
+.i-amphtml-story-subscriptions-dialog-banner {
+  position: relative !important;
+  width: 100% !important;
+  border-radius: 16px 16px 0px 0px !important;
+  font-size: 24px !important;
+  text-align: center !important;
+  height: 44px !important;
+  background: #EDEDED !important;
+}
+
+.i-amphtml-story-subscriptions-dialog-content {
+  position: relative !important;
+  margin: 28px 28px 24px 28px !important;
+}
+
+.i-amphtml-story-subscriptions-publisher-text {
+  position: relative !important;
+  margin-bottom: 41px !important;
+}
+
+.i-amphtml-story-subscriptions-price, .i-amphtml-story-subscriptions-title,
+.i-amphtml-story-subscriptions-subtitle-first, .i-amphtml-story-subscriptions-subtitle-second {
+  overflow: hidden !important;
+  white-space: nowrap;
+  text-align: center;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.i-amphtml-story-subscriptions-price {
+  width: 100% !important;
+  font-size: 24px !important;
+  margin-top: 28px !important;
+  letter-spacing: -0.4px !important;
+  height: 41px !important;
```

Comment:
Here and below, how does it look if the text goes multi lines? Please test it and make sure your layout adapts correctly

Reply:
If the length goes over than the width, we simply omit it for now: https://screenshot.googleplex.com/BBzXzHRac9nMsr6.